packages/connector/src/snap.ts

```diff
@@ -251,6 +251,56 @@ async function removeTrustedDapp(
   );
 }
 
+/**
+ * Adds a dapp to the settings list.
+ *
+ * @return Result<boolean> - true if the addition was successful
+ */
+async function addDappSettings(
+  this: Masca,
+  origin: string
+): Promise<Result<boolean>> {
+  return sendSnapMethod(
+    this,
+    { method: 'addDappSettings', params: { origin } },
+    this.snapId
+  );
+}
+
+/**
+ * Removes a dapp from the settings list.
+ *
+ * @return Result<boolean> - true if the addition was successful
+ */
+async function removeDappSettings(
+  this: Masca,
+  origin: string
+): Promise<Result<boolean>> {
+  return sendSnapMethod(
+    this,
+    { method: 'removeDappSettings', params: { origin } },
+    this.snapId
+  );
+}
+
+/**
+ * Removes queryCredentials permissions for the selected dApp. Popups will appear when the dApp tries to query VCs.
```

Comment:
Change the comment to describe the abstract function (currently supports only `queryCredentials` but will support other methods in the future).